--- D:/Develop/de.evil-co.cms/de.evil-co.cms/tar/files/lib/acp/page/IndexPage.class.php	Sat Nov 06 16:32:05 2010
+++ D:/Develop/de.evil-co.cms/IndexPage_patched.class.php	Sun Nov 21 23:22:30 2010
@@ -97,6 +97,25 @@
 		if (WCF::getUser()->getPermission('admin.system.package.canUpdatePackage')) {
 			require_once(WCF_DIR.'lib/acp/package/update/PackageUpdate.class.php');
 			$this->updates = PackageUpdate::getAvailableUpdates();
+			
+			// kick wbb 3.0 updates
+			foreach ($this->updates as $packageID => $package) {
+				if ($package['package'] == 'com.woltlab.wbb') {
+					foreach ($package['versions'] as $version => $packageVersion) {
+						if (Package::compareVersion($version, '3.0.0 Beta 1', '>=')) {
+							unset($this->updates[$packageID]['versions'][$version]);
+						}
+					}
+					
+					if (!count($this->updates[$packageID]['versions'])) {
+						$this->updates = PackageUpdate::getAvailableUpdates(false);
+						unset($this->updates[$packageID]);
+					}
+					else {
+						$this->updates[$packageID]['version'] = end($this->updates[$packageID]['versions']);
+					}
+				}
+			}
 		}
 	}
 	
